src/qt/locale/bitcoin_it.ts

```diff
@@ -3118,52 +3118,52 @@ p, li { white-space: pre-wrap; }
     </message>
     <message numerus="yes">
         <source>Open for %n more block(s)</source>
-        <translation type="unfinished">
-            <numerusform></numerusform>
-            <numerusform></numerusform>
+        <translation>
+            <numerusform>Aperta per %n altro blocco</numerusform>
+            <numerusform>Aperta per altri %n blocchi</numerusform>
         </translation>
     </message>
     <message>
         <source>Open until %1</source>
-        <translation type="unfinished"></translation>
+        <translation>Aperta fino a: %1</translation>
     </message>
     <message>
         <source>conflicted with a transaction with %1 confirmations</source>
-        <translation type="unfinished"></translation>
+        <translation>Incontrati conflitti con una transazione con %1 conferme</translation>
     </message>
     <message>
         <source>in memory pool</source>
-        <translation type="unfinished"></translation>
+        <translation>nella riserva di memoria</translation>
     </message>
     <message>
         <source>not in memory pool</source>
-        <translation type="unfinished"></translation>
+        <translation>non nella riserva di memoria</translation>
     </message>
     <message numerus="yes">
         <source>, broadcast through %n node(s)</source>
-        <translation type="unfinished">
-            <numerusform></numerusform>
-            <numerusform></numerusform>
+        <translation>
+            <numerusform>, trasmessa attraverso %n nodo</numerusform>
+            <numerusform>, trasmessa attraverso %n nodi</numerusform>
         </translation>
     </message>
     <message numerus="yes">
         <source>matures in %n more block(s)</source>
```

Comment:
cosa si intende per matures?

Reply:
E' un termine tecnico, letteralmente tradotto con "maturare" in Italiano. Indica il momento (dopo un certo di numero di blocchi) in cui una transazione puo' essere confermata. This might help: https://bitcoin.stackexchange.com/questions/23957/transaction-maturity-vs-block-maturity

I think the translation is correct, as it relies on the same metaphor - the one around maturity - used for the english counterpart.